CONTRIBUTORS.txt

```diff
@@ -111,28 +127,35 @@ needs to be listed here.
 - Rémi Doreau <remi.d45@gmail.com>
 - Saleem Abdulrasool <compnerd@compnerd.org>
 - Sebastian Thiebaud <770074+SebastianThiebaud@users.noreply.github.com>
+- Sebastian Vogt <sebastian1210@web.de>
 - Semen Zhydenko <simeon.zhidenko@gmail.com>
 - Sergej Jaskiewicz <jaskiewiczs@icloud.com>
 - Sergo Beruashvili <beruashvili@gmail.com>
 - Shaun Hubbard <shaunhubbard2013@icloud.com>
 - Shekhar Rajak <shekhar_rajak@apple.com>
+- Si Beaumont <beaumont@apple.com>
 - Siemen Sikkema <s.h.sikkema@gmail.com>
 - Simon Evans <si@si.org>
+- Stepan Ulyanin <99296376+stepan-ulyanin@users.noreply.github.com>
```

Comment:
@stepan-ulyanin  if you would like to have a real email address here, please add a comment with your preferred address. Thank you!

Reply:
Hey! We can use my Apple address -> sulyanin@apple.com. Thanks!